tests/test_move.py

```diff
@@ -1,16 +1,13 @@
 from engine.move import Move
-from engine.gamestate import GameState
-from engine.board import Board
-from engine.card import Card
-from engine.cardstack import CardStack
 
 import pytest
 import unittest
-
-from mock import create_autospec
+import util
 
 
+# TODO: Are some of these integration tests?
```

Comment:
I felt like many of these tests check interactions between `Move` and `GameState` (so aren't unit tests). Might be helpful to include them in an integration test package at some point in the future.